Lesson_5/Lesson_5/Scanner.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Lesson_5
+{
+    public interface IScanner
+    {
+        Task<ScanResultModel> Scan();
+    }
+
+    public class Scanner : IScanner
+    {
+        public async Task<ScanResultModel> Scan()
+        {
+            var ScanResult = new ScanResultModel();
+
+            using (FileStream FileStream = File.OpenRead("image.jpg"))
+            {
+                var Bytes = new byte[FileStream.Length];
+                await FileStream.ReadAsync(Bytes, 0, Bytes.Length);
+                ScanResult.ScannedDocument = Bytes;
+            }
+
+            ScanResult.CPUUsage = new Random().Next(2 ^ 10 * 10, 2 ^ 10 * 50);
+            ScanResult.HDDUsage = new Random().Next(2 ^ 10 * 100, 2 ^ 10 * 500);
```

Comment:
При таком подходе Вы рискуете получить два одинаковых "случайных числа". Либо в классе создавайте статический объект Random, либо если это .NET6
```suggestion
            ScanResult.CPUUsage = Random.Shared.Next(2 ^ 10 * 10, 2 ^ 10 * 50);
            ScanResult.HDDUsage = Random.Shared.Next(2 ^ 10 * 100, 2 ^ 10 * 500);
```

Reply:
Когда создаётся объект Random, то его внутреннее начальное состояние определяется по текущему значению микросекунд времени ОС. Если подряд создать два объекта Random, то они будут одинаковыми.
Да и для генерации двух случайных чисел создавать каждый раз два(!) объекта Random в памяти - зачем злить Сборщика мусора?